docs/fhir-resources/valueset-validate-code.md

```diff
@@ -11,3 +11,50 @@ http://localhost:8080/fhir/ValueSet/gps/$validate-code?coding=http://snomed.info
 
 #### Validate a code against an implicit valueset expanded against a specific SNOMED release - alternative format
 [http://localhost:8080/fhir/ValueSet/$validate-code?codeSystem=http://snomed.info/sct/900000000000207008/version/20190731&url=http://snomed.info/sct?fhir_vs=ecl/<<34014006 |Viral disease|&code=840539006](http://localhost:8080/fhir/ValueSet/$validate-code?codeSystem=http://snomed.info/sct/900000000000207008/version/20190731&url=http://snomed.info/sct?fhir_vs=ecl/%3C%3C34014006%20|Viral%20disease|&code=840539006)
+
+## POST Method Request Example(s)
+
+```
+curl --request POST 'http://localhost:8080/fhir/ValueSet/$validate-code' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "resourceType": "Parameters",
+    "parameter": [
+        {
+             "name" : "coding",
+             "valueCoding": {
+                "code": "46261000087101",
+                "system": "http://snomed.info/sct"
+              }
+        },
+        {
+            "name": "valueSet",
+            "resource": {
+            	"resourceType": "ValueSet",
+                "compose": {
+                    "include": [
+                        {
+                            "system" : "http://snomed.info/sct",
+                            "filter": [
+                                {
+                                    "property": "concept",
+                                    "op": "is-a",
+                                    "value": "21351003"
+                                }
+                            ]
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "name": "system-version",

```

Comment:
I don't know why GitHub is saying this PR is going to merge cleanly; I know I've made two updates to this json here at what I'm seeing here is an earlier version.

Well, hopefully I'll get a merge conflict and I'll be able to pick.   I might just save off a branch first in case I need to refer back...